server/controller/getForms.js

```diff
@@ -0,0 +1,13 @@
+const query = require('../database/queries/getForms');
+exports.getForms = (req, res) => {
+	const employeeID = req.params.employeeID;
+	query(employeeID)
+		.then((data) => {
+			if (data.length != 0) {
+				res.json(data);
+			} else {
+				res.status(404).json({ message: ' Employee form Not Found' });
```

Comment:
Let`s say "The Employee didn`t review/evaluate yet!"

Reply:
done !